sideboard/lib/_threads.py

```diff
@@ -1,22 +1,25 @@
 from __future__ import unicode_literals
+import sys
 import time
-import platform
-import psutil
 import heapq
-import prctl
 import ctypes
-import sys
+import platform
 import traceback
-from sideboard.debugging import register_diagnostics_status_function
+import threading
 from warnings import warn
 from threading import Thread, Timer, Event, Lock
 
+import six
 from six.moves.queue import Queue, Empty
 
-from sideboard.lib import log, on_startup, on_shutdown
+from sideboard.lib import log, config, on_startup, on_shutdown
+from sideboard.debugging import register_diagnostics_status_function
 
-
-import threading
+try:
+    import prctl
+    import psutil
+except ImportError:
+    prctl = psutil = None  # For platforms without this support.
```

Comment:
See https://github.com/magfest/sideboard/issues/25 for an explanation of why we're making these conditional imports.